src/Query/SqlWalker.php

```diff
@@ -1352,6 +1352,9 @@ public function walkSelectExpression(AST\SelectExpression $selectExpression): st
                 $sql .= $this->walkNewObject($expr, $selectExpression->fieldIdentificationVariable);
                 break;
 
+            case $expr instanceof AST\AllFieldsExpression:
+                throw new LogicException('All fields Expression are not supported here.');
```

Comment:
Will users be able to understand the issue? `here` may be confusing without more context.